Assets/Scripts/Models/World.cs

```diff
@@ -632,7 +632,7 @@ void LoadInventoryPrototypesFromFile(string inventoryXmlText)
         //furniturePrototypes["Door"].RegisterUpdateAction( FurnitureActions.Door_UpdateAction );
         //furniturePrototypes["Door"].IsEnterable = FurnitureActions.Door_IsEnterable;
 
-            //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
+        //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
```

Comment:
Yes but after 12 hours straight coding I really don't have the patience to fix every style error that was preexisting in code I've touched.

I hope you'll pardon me, but I am entirely too tired and mind-fried to handle your comments in a polite tone to match your own.